obspy/io/sac/sactrace.py

```diff
@@ -1100,7 +1100,7 @@ def reftime(self, new_reftime):
     # --------------------------- I/O METHODS ---------------------------------
     @classmethod
     def read(cls, source, headonly=False, ascii=False, byteorder=None,
-             checksize=False, debug_strings=False):
+             checksize=False, debug_strings=False, **kwargs):
```

Comment:
That's actually easier for me to code. I was trying to keep all the code unchanged  until absolutely needed. If you guys have no problem with adding a new parameter to `read` in `obspy.io.sac.sactrace` and change the calling statement in `_internal_read_sac` in `obspy.io.sac.core`, I will be more than happy to do that.